x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -275,7 +271,7 @@ export class TaskManagerPlugin
         taskStore.aggregate(opts),
       get: (id: string) => taskStore.get(id),
       remove: (id: string) => taskStore.remove(id),
-      bulkRemoveIfExist: (ids: string[]) => bulkRemoveIfExist(taskStore, ids),
```

Comment:
Replacing `bulkRemoveIfExists` with `bulkRemove` since the error path within `x-pack/plugins/task_manager/server/lib/bulk_remove_if_exist.ts` would never get reached in a bulk request (you have to loop the responses, no errors thrown in 404 scenario). I added code to handle 404 within `x-pack/plugins/alerting/server/rules_client/common/try_to_remove_tasks.ts`.